WasatchNET/EEPROM.cs

```diff
@@ -295,6 +295,34 @@ public float[] wavecalCoeffs
 
         float[] _wavecalCoeffs;
 
+
+        public float[] intensityCorrectionCoeffs
+        {
+            get { return _intensityCorrectionCoeffs; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _intensityCorrectionCoeffs = value;
+                handler?.Invoke(this, new EventArgs());

```

Comment:
I might log a warning if set to an array whose length is other than _intensityCorrectionCoeffs.length + 1